manifests/virtualenv.pp

```diff
@@ -31,29 +31,30 @@
 #  }
 #
 define python::virtualenv (
-  $ensure                          = 'present',
-  $version                         = 'system',
-  $requirements                    = false,
-  $systempkgs                      = false,
-  $venv_dir                        = $name,
-  $ensure_venv_dir                 = true,
-  $distribute                      = true,
-  $index                           = false,
-  $owner                           = 'root',
-  $group                           = 'root',
-  $mode                            = '0755',
-  Optional[Stdlib::HTTPUrl] $proxy = undef,
-  $environment                     = [],
-  $path                            = ['/bin', '/usr/bin', '/usr/sbin', '/usr/local/bin'],
-  $cwd                             = undef,
-  $timeout                         = 1800,
-  $pip_args                        = '',
-  $extra_pip_args                  = '',
-  $virtualenv                      = undef,
+  $ensure          = 'present',
+  $version         = 'system',
+  $requirements    = false,
+  $systempkgs      = false,
+  $venv_dir        = $name,
+  $ensure_venv_dir = true,
+  $distribute      = true,
+  $index           = false,
+  $owner           = 'root',
+  $group           = 'root',
+  $mode            = '0755',
+  $proxy           = undef,
+  $environment     = [],
+  $path            = ['/bin', '/usr/bin', '/usr/sbin', '/usr/local/bin'],
+  $cwd             = undef,
+  $timeout         = 1800,
+  $pip_args        = '',
+  $extra_pip_args  = '',
+  $virtualenv      = undef,
```

Comment:
this is a public interface, so it should have types

Reply:
Ah.. forget this :( sorry.

Reply:
Fixed.